apps/researcher/src/app/[locale]/(objects)/objects/[id]/actions.ts

```diff
@@ -40,23 +41,27 @@ export async function removeObjectFromList(id: number, communityId: string) {
 interface AddUserEnrichmentProps {
   description: string;
   citation: string;
-  about: string;
-  attributionId: string;
+  inLanguage: string;
```

Comment:
To be sure: `inLanguage` is optional in the `enricher` package.

Reply:
I have now made it optional in the code, however, there is no empty option in the LanguageSelector component, I will discuss with Bas if we need to add this.